### DISCUSSION TITLE
GitHub Actions is currently disabled for your account. Please reach out to GitHub Support for assistance.

### DISCUSSION THREAD

**Discussion by IAVARABBASOV** [community/community#140982] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Show & Tell

### Body

I can't find any restriction reason in my repositories, I demand to enable Actions for my Account Please.

Your previous message: 
( In order to clear the restrictions from your account we'll need you to:
 
double-check and remove any remaining offending repositories
agree to follow our [Terms of Service](https://help.github.com/articles/github-terms-of-service) going forward, you can do this by simply typing a sentence saying as much
 
Let us know when you've completed these steps and we'll be happy to review your account again.
 
Best regards,
Charlie )

![image](https://github.com/user-attachments/assets/34c80258-8b1d-4fa2-a636-70169aafb026)

**Comment by ebndev** [comment, marked as answer]
Hi @IAVARABBASOV,

This community is managed by GitHub employees, but it is not a support forum. The only place to go for answers to account-specific questions like this one is opening a ticket on our [Support page](https://support.github.com/contact).

This is a public forum, and the community is unable to assist with nor escalate your ticket, but we can assure you it is in the right place. It will be answered in the order it was received based on our team's resources and the amount of tickets ahead of yours.

As no other Community Discussions team members will be able to provide additional help with account related questions I am going to close this discussion. We appreciate your understanding!
